src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -3430,6 +3425,7 @@ boolean creativeRulesApply(Location location)
 
     public String allowBuild(Player player, Location location)
```

Comment:
Had forgotten about this. I get that this allows plugins a shortcut that may make checks a little bit faster, but it feels like poor design. A lot of the overloads also feel clunky. If we're making breaking changes by removing deprecated claim methods (in v17 or 20 or whenever) I think these should be candidates or at very least certain overloads should be removed.

Reply:
For clarity, I'd like to deprecate some or all of these methods now so that they can be removed after a major version bump.

Reply:
It is poor design I agree, I think this is remnants of bigscary's original design.

Reply:
It does need some sort of equivalent as it handles the wilderness features, but the base method isn't great. If you're on board with a rework of this area I'll brainstorm a bit tonight.